README.md

```diff
@@ -69,13 +61,31 @@ The Cypress application will open up and it should be self evident how to run th
 
 _Our CI/CD is setup via Travis—you can find its configuration at `.travis.yml`_
 
-## Verify that the Design System works within CMS
+## Usage
+
+To use a release downstream, update your app's `package.json` as follows:
```

Comment:
I don't know if we should mention, but for monorepo, if one team sets EDS to `v1.2.3` and another team needs a `v1.2.4`, the first team will have to deal with the fact that EDS will get bumped. So, `v1.2.3` should not be considered as a lock for that project, but a convenient means to avoid keeping track of exact sha1's

Reply:
Not sure what you mean. Seems like CMS and monorepo could be on different versions.